docs/layers/monitoring/datadog/datadog.mdx

```diff
@@ -207,111 +280,40 @@ Additionally, SLAs are typically on a calendar year, and not a rolling window of
 - Aggregate as much as possible into an SLO, but for the _right_ audience
 
-#### Erik’s Theory
-
-- SLOs are a simple way to give a team responsible for something know when they aren’t meeting their objectives, as much as they are valuable to provide quality of service to business end-users.
-
-- We can _solve_ the practical implementation of SLOs once for all of our customers, while giving each customer the ability to tailor it to their needs
-
-- SLOs should be thought about like an organization structure, where each node in the org has an SLO
-
-- SLOs where created to support managing SLAs, but nowadays should be valuable to any team. But maybe we should then name them something else, so we don’t trigger a lot of associations with what SLOs are to other people
-
-- SLOs can be valuable to validate our assumptions on how stable some system is (@Jeremy Grodberg 's point on KMS keys)
-
-- SLOs might not have urgent action items, but if they are not urgent, then what should we do? Call them something else?
-
-- SLOs convey the institutional knowledge of what is healthy in a unit everyone understands (percentages) @Jeremy Grodberg
-
-- SLOs convey the health of the system for some period of time @Matt Calhoun
-
-- SLOs should/can also be visualized over time as any other metric
-
-- Bad SLOs can be detrimental, but that only means our objectives were wrong. But having the objectives is what is so valuable, because they can be corrected. We should refine/correct our objectives until they are accurate.
-
-- When SLOs are violated, it might be that the objective was wrong, rather than the systems are failing
-
-- We can boil the ocean with SLOs, but we dont' want to or need to.
-
-- “If a tree falls in a forest and no one was listening does it make a sound?”
-
-- We can have SLOs that are only visible to some, and don’t impact other teams
-
-- It helps us move from “feelings” of why something is violated to concrete explanation
-
-- There are categories, classes or levels of SLOs
-
-- e.g. Capacity Planning SLOs
-
-- e.g. Critical SLOs (or SLA SLOs)
-
-- e.g.  User Journey SLOs [https://cloud.google.com/blog/products/management-tools/practical-guide-to-setting-slos](https://cloud.google.com/blog/products/management-tools/practical-guide-to-setting-slos) - measures a set of interactions a user has with a service to achieve some end result.
-
-- What’s interesting about this kind of SLO, is that it measures an outcome that was influenced by possibly dozens of underlying services
-
-- A system of SLOs should help us identify cascading failures before it’s too late, as the underlying SLOs should fire before higher-level SLOs are affected
-
-- While some might say this is at odds with setting different thresholds on the top-level SLO, I disagree - because the top-level SLOs are/should be the slowest to move as a result of HA/redundancies within the system. It’s when those redundancies are giving out, we need to be very afraid.
```

Comment:
Please note I removed "Erik's Theory" we can re-add this at a later point after it has been refined